hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/web/TestDatanodeHttpXFrame.java

```diff
@@ -69,7 +68,7 @@ public void testNameNodeXFrameOptionsDisabled() throws Exception {
     cluster = createCluster(xFrameEnabled, null);
     HttpURLConnection conn = getConn(cluster);
     String xfoHeader = conn.getHeaderField("X-FRAME-OPTIONS");
-    Assert.assertTrue("unexpected X-FRAME-OPTION in header", xfoHeader == null);
+    Assert.assertNull(xfoHeader);
```

Comment:
```suggestion
    Assert.assertNull("unexpected X-FRAME-OPTION in header", xfoHeader);
```

Reply:
@jojochuang Thank you very much for helping to review the code, I will modify the code!